usr/lib/xtable/Makefile

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+# SPDX-License-Identifier: MIT OR GPL-2.0-only
+
+LUAXTABLE_MODULE ?= test
```

Comment:
I would let it raise an error if not defined.. perhaps you can just say how it should be used in the error message